core/src/main/java/org/jdbi/v3/core/mapper/EnumMapper.java

```diff
@@ -13,17 +13,14 @@
  */
 package org.jdbi.v3.core.mapper;
 
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
-
-import org.jdbi.v3.core.statement.StatementContext;
-
 /**
  * Column mapper for Java {@code enum} types.
  * @param <E> the enum type mapped
+ *
+ * @deprecated there is no reason for this to be API
  */
+@Deprecated
+// TODO jdbi4: delete
 public abstract class EnumMapper<E extends Enum<E>> implements ColumnMapper<E> {
```

Comment:
This class is still the only access point for `EnumMapper.byName()` and `EnumMapper.byOrdinal()`, e.g. for use with `ResultBearing.map(ColumnMapper<T>): ResultIterable<T>`